### PR TITLE
Drawer design review changes

### PR DESCRIPTION
**Pull request by jyothishnt**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1328

## Description
design review changes for GB drawer

## Deployment URL
http://drawer-design-update.review.ensembl.org/

## Views affected
GB - drawer - Gene and transcript summary

### Other effects
N/A

- [ ] I have checked any requirements for Google Analytics
- [ ] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

## Possible complications
NA

### CODE REVIEW

File: src/ensembl/src/shared/components/view-in-app/ViewInApp.scss [hunk 1]
```diff
@@ -1,7 +1,8 @@
 @import 'src/styles/common';
 
 .label {
-  color: $grey;
+  color: $black;
```
Comment (azangru) [inline src/ensembl/src/shared/components/view-in-app/ViewInApp.scss:4]: Species page right now:

image

File: src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx [hunk 2]
```diff
@@ -165,18 +181,18 @@ const GeneSummary = () => {
...
       </div>
 
       <div className={classNames(rowClasses, styles.downloadRow)}>
         <div className={styles.value}>
-          <div
-            className={styles.downloadLink}
+          <ShowHide
+            label="Download"
+            isExpanded={shouldShowDownload}
             onClick={() => showDownload(!shouldShowDownload)}
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx:194]: - No longer need the close button
- Should InstantDownload component appear to the right of the ShowHide, as it does now, or below it?

File: src/ensembl/src/shared/components/layout/StandardAppLayout.scss [hunk 3]
```diff
@@ -146,6 +146,6 @@ $drawerWindowWidth: 45px;
 
 .drawerClose {
   position: absolute;
-  right: calc(#{$drawerWindowWidth} + 10px);
-  top: 10px;
+  right: calc(#{$drawerWindowWidth} + 30px);
```
Comment (azangru) [inline src/ensembl/src/shared/components/layout/StandardAppLayout.scss:149]: Are you also seeing the close button right over the scrollbar now?

File: src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx (hunk 2 above)
Comment (azangru) [reply src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx:194]: As a reminder: also use ShowHide in TranscriptSummary component.

File: src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss [hunk 4]
```diff
@@ -69,3 +91,8 @@
   right: 0;
   top: 0;
 }
+
+.darkText {
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss:95]: `.darkText` -> `.viewInAppLabel` (to make the class name more meaningful)

File: src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss [hunk 5]
```diff
@@ -69,3 +91,8 @@
...
+  color: $black;
+  font-weight: 300;
+}
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss:98]: Empty line?

File: src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx [hunk 6]
```diff
@@ -29,12 +29,14 @@ import {
...
 } from 'src/shared/state/ens-object/ensObjectHelpers';
+import { pluralise } from 'src/shared/helpers/formatters/pluralisationFormatter';
 import { getBrowserActiveEnsObject } from 'src/content/app/browser/browserSelectors';
 
 import ExternalReference from 'src/shared/components/external-reference/ExternalReference';
 import InstantDownloadGene from 'src/shared/components/instant-download/instant-download-gene/InstantDownloadGene';
 import ViewInApp from 'src/shared/components/view-in-app/ViewInApp';
-import CloseButton from 'src/shared/components/close-button/CloseButton';
+import ShowHide from 'src/shared/components/show-hide/ShowHide';
+import QuestionButton from 'ensemblRoot/src/shared/components/question-button/QuestionButton';
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx:39]: 🙅‍♂️ ensemblroot

File: src/ensembl/src/content/app/browser/drawer/drawer-views/transcript-summary/TranscriptSummary.tsx [hunk 7]
```diff
@@ -40,15 +41,15 @@ import { getBrowserActiveEnsObject } from 'src/content/app/browser/browserSelect
 import { InstantDownloadTranscript } from 'src/shared/components/instant-download';
 import ViewInApp from 'src/shared/components/view-in-app/ViewInApp';
 import ExternalReference from 'src/shared/components/external-reference/ExternalReference';
-import CloseButton from 'src/shared/components/close-button/CloseButton';
 import QuestionButton from 'ensemblRoot/src/shared/components/question-button/QuestionButton';
```
Comment (azangru) [inline src/ensembl/src/content/app/browser/drawer/drawer-views/transcript-summary/TranscriptSummary.tsx:44]: 🙅‍♂️ ensemblRoot

File: src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss (hunk 4 above)
Comment (jyothishnt) [reply src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss:95]: Infact I used the same initially, but then found there was a classname .lightText in TranscriptSummary.scss and thought we may not want it to look like app specific. Also I saw in one of your comments in a PR

**Review by azangru — APPROVED** [review]
Code looks good; the deployment also looks good; but I have not tried to compare the actual changes with what's in the ticket 😃

**Comment by jyothishnt** [comment]
That could be done during the QA :-)

**Pull request merged**